src/server/accounts.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "_id": "admin",
+    "bcryptPassword": "$2b$10$o78d5eRDsumu7L.Uq7WlQO2imW/9k7VWateMa/4YOu7Yxad4yF15m",
```

Comment:
We probably shouldn't include a default password. Can we remove this file?

Reply:
That's the admin user created on server startup. I can add this file to git ignore and make it to be created if not exists